### DISCUSSION TITLE
'Numpad enter' doesn't work with `Select` component

### DISCUSSION THREAD

**Discussion by hunterwilhelm** [mantinedev/mantine#1579] [Feedback]
Hello, my users are complaining that when they are done typing in a `Select` element, the 'Numpad enter' button doesn't work. I tried it out on the website here: https://mantine.dev/core/select/ (v4.2.8) and the issue is present.

I followed the same steps with some non-Mantine searchable selects and they worked fine with the "Numpad enter" so that key isn't broken on my machine or configuration.

https://user-images.githubusercontent.com/71348224/172246640-765a2edd-8102-444a-8ec5-8f5e12fc9318.mov

**Comment by rtivital** [comment]
The issue will be resolved in 5.0 release
